src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js/big';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const x = new Big(numberOne);
+  let result;
+
+  switch (operation) {
+    case '-':
+      result = x.minus(numberTwo);
+      break;
+    case '+':
+      result = x.plus(numberTwo);
+      break;
+    case '*':
+      result = x.times(numberTwo);
+      break;
+    case '/':
+      result = x.div(numberTwo);
+      break;
```

Comment:
dont forget to manage DIVISION BY ZERO on next milestone else, it will generate an error